index.bs

```diff
@@ -716,6 +716,21 @@ partial interface Gamepad {
 <dfn attribute for="Gamepad" id="gamepad-getvrdisplays-attribute">displayId</dfn>
 Return the {{VRDisplay/displayId}} for the associated {{VRDisplay}}.
 
+# Processing Model # {#proc}
+
+## submitFrame ## {#submitframe-proc}
+
+There are multiple situations where applications won't invoke {{VRDisplay}}.{{submitFrame()}} at the {{VRDisplay}} native framerate. The User Agent MUST intervene to protect the user from motion sickness or consistently inform her about different situations, such as errors or navigation between pages. This is the breakdown of the different scenarios:
+
+* The application calls {{VRDisplay}}.{{submitFrame()}} at the expected rate, but only a few frames are dropped. The User Agent MUST re-project the last submitted frame.
+
+* The application calls {{VRDisplay}}.{{submitFrame()}} below the expected framerate to a degree that could cause discomfort to the user (simulator sickness). The User agent SHOULD render a trusted interstitial that obscures or minimize the application until a consistent, acceptable frame rate is achieved. If the low framerate persists for an extended period of time the User Agent MUST communicate the user that the application is too slow to be presented and the User Agent MAY terminate presentation.
```

Comment:
Are there any plans to [report the expected framerate](https://github.com/w3c/webvr/issues/37)?